config/routes.rb

```diff
@@ -185,7 +185,7 @@
 
     # provider/admin
     namespace :admin do
-      resource :quickstarts, only: :show
+      resource :quickstarts, only: :show if ThreeScale.saas?
```

Comment:
This is not necessary, because there is https://github.com/3scale/porta/blob/master/app/controllers/provider/admin/quickstarts_controller.rb#L15

I don't know why this way was chosen over condition on the route initially. I think maybe it's to enable the path helpers (i.e. `provider_admin_quickstarts_path`).

In any case, from the user's point of view it's pretty much the same - they'll receive a 404.